### PR TITLE
Update supported build scan plugin to 2.1-rc-1.

### PR DESCRIPTION
**Pull request by felipefzdz**

### CODE REVIEW

File: subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy [hunk 1]
```diff
@@ -25,6 +25,7 @@ import static org.gradle.internal.scan.config.BuildScanPluginCompatibility.MIN_S
 class BuildScanPluginSmokeTest extends AbstractSmokeTest {
 
     private static final List<String> GRACEFULLY_UNSUPPORTED_WITHOUT_FAILURE = [
+            "2.0.2",
```
Comment (wolfs) [inline subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy]: Plugin 2.0.2 is still support with Gradle 5.1, right?

Comment (guylabs) [reply subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy]: Yes. The change has been reverted.

**Comment by guylabs** [comment]
About the file input snapshots: we will have to observe the consequences in terms of payload size, upload time and decide whether to always capture it (given it takes a lot of extra disk space)

**Comment by wolfs** [comment]
That means a separate PR, right. And yes, I always want to capture them so I can analyze cache misses...

**Comment by etiennestuder** [comment]
@wolfs so every build published by everyone will have the feature enabled? if so, this is against what we recommend to our users. the disk space usage will go up significantly, scan upload might also be affected.

**Comment by wolfs** [comment]
What is the recommendation? Which builds should have the feature enabled?

**Comment by ldaley** [comment]
@wolf all builds where the slow connection property isn’t set.

**Comment by ldaley** [comment]
We recommend to have it on for all builds, unless you have a larger build and bad network connection.

**Comment by wolfs** [comment]
Sounds good!

**Comment by wolfs** [comment]
Follow up for enabling individual input file differences: https://github.com/gradle/gradle/pull/7964

**Pull request merged**